python/python/res/enkf/config/enkf_config_node.py

```diff
@@ -115,6 +116,25 @@ def create_ext_param(cls, key, key_list, output_file = None):
         keys = StringList( initial = key_list )
         return cls._alloc_ext_param_node(key, keys, output_file)
 
+
+    # This method only exposes the details relevant for Everest usage.
+    @classmethod
+    def create_gen_data(cls, key, file_fmt, report_steps = None):
+        active_steps = IntVector( )
+        if report_steps is None:
```

Comment:
Is the default value of `active_steps` to be `IntVector([1])`? In that case I think you should reflect this by the setting the default value of `report_steps` to be `[1]`.

Reply:
> Is the default value of active_steps to be IntVector([1])?

Yes.

> In that case I think you should reflect this by the setting the default value of report_steps to be [1].

Well - I am under the impression that it is bad form to use mutable objects as default values in initialization calls?

Reply:
That is a good point! It would also be quite bad if the method actually mutate `report_steps`. But, it is better to be safe than sorry, so how about `report_step=(1)`. Tuples for the win!?

Reply:
OK - tuple coming.